Add release notes for 0.14.5 (#944)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

+++
<!-- markdownlint-disable no-duplicate-header -->

## v0.14.5

This is a bugfix release:

* The `subctl gather` command now collects iptables information for OVN-Kubernetes CNI.
* Fixed issue while running `subctl gather` command for OVN-Kubernetes CNI.

> * Fixed issue when Gateway pod restart causing data path disruption.

Reviewer: ```suggestion
* Fixed issue where a Gateway pod restart caused data path disruption.
```

> * Fixed issue of IPsec pluto process crashes when the remote endpoint is unstable.

Reviewer: ```suggestion
* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
```
